src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -5042,6 +5042,277 @@ class StubGenerator: public StubCodeGenerator {
     return (address) start;
   }
 
+  void adler32_process_bytes(Register buff, Register s1, Register s2, VectorRegister vtable,
+    VectorRegister vzero, VectorRegister vbytes, VectorRegister vs1acc, VectorRegister vs2acc,
+    Register temp0, Register temp1, Register temp2,  Register temp3,
+    VectorRegister vtemp1, VectorRegister vtemp2, int step, Assembler::LMUL LMUL) {
```

Comment:
Better to use small ‘lmul’ and `lmulx2` for `LMUL` and `LMULx2` respectively to be consistent in naming style.

Reply:
Done